monitors/README.md

```diff
@@ -1,4 +1,32 @@
-# Monitors *(Windows Only)*
+# New End of Run Monitor *(Any Platform)*
+
+This End of Run Monitor can be run on any platform either standalone on the command line or
+as a scheduled task. All it requires is a CSV file of the following format:
+
+```
+WISH,1234,wish_lastrun.txt,wish_summary.txt,/archive/wish/data,.nxs
```

Comment:
Is there a reason we are using a CSV file here and not a settings.py file? This seems like it would be more inline with the rest of the code base

Reply:
So that the last run can be updated each time a new run arrives, this would be much more difficult to do in a settings.py file. If I just stored the last run number in a text file then we would have the issue of matching up keys across two files, which I think is more complicated than just storing everything in CSV.